ui/app/models/pki/role.js

```diff
@@ -287,6 +270,11 @@ export default class PkiRoleModel extends Model {
           docLink: '/api-docs/secret/pki#allowed_domains',
         },
       },
+      'Key parameters': {
```

Comment:
Removed `<FormFieldLabel>` from the component template below, so parent is responsible for rendering toggle group-specific text